www/AppReview.js

```diff
@@ -35,6 +35,7 @@ exports.openStoreScreen =
  */
 function(packageName) {
```

Comment:
Let's move `writeReview` to be function argument and be falsy by default. This flag will work only for iOS
```suggestion
function(packageName, writeReview) {
```
Also don't forget to update jsdocs comment with param `writeReview` description.